orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/config/RedisOrcaQueueConfiguration.kt

```diff
@@ -85,4 +87,7 @@ class RedisOrcaQueueConfiguration : RedisQueueConfiguration() {
   ) =
     RedisPendingExecutionService(jedisPool, mapper)
 
+  @Bean
+  @ConditionalOnMissingBean(DynamicConfigService::class)
+  internal fun springTransientConfigService(): DynamicConfigService = NOOP
```

Comment:
Had to add this to get the integration tests in `orca-queue-redis` to pass. I'm not sure where this was coming from before or whether there was a reason it wasn't necessary.